src/Spinners/LoadSpinnerText.tsx

```diff
@@ -0,0 +1,41 @@
+import React, {
+  FunctionComponent, ReactElement, useContext,
+} from 'react';
+import styled, { ThemeContext } from 'styled-components';
+import { BaseTheme } from '../Theme';
+
+export interface LoadSpinnerTextProps {
+  /** Specifies the text under the spinner */
+  children: string;
+  /** Specifies the color of the text under the spinner */
+  color?: string;
+}
+
+const StyledLoadSpinnerText = styled.div`
+  background: transparent;
+  color: ${({ color }): string => (color)};
+  font: ${({ theme }): string => (theme.font.heading)};
+  margin-top: ${({ theme }): string => (theme.ws.large)};
+  margin-bottom: ${({ theme }): string => (theme.ws.medium)};
```

Comment:
I would leave out this `bottom-margin`, to provide more flexibility in how the spinner is displayed within the parent component.